docs/_markbind/navigation/mainNav.md

```diff
@@ -0,0 +1,52 @@
+<navigation>
+
+* <span class="lead">**User Guide**</span>
+    * [Getting Started]({{baseUrl}}/UserGuide.html#getting-started)
+    * [Interpreting the Report]({{baseUrl}}/UserGuide.html#interpreting-the-report)
+      * [Chart Panel]({{baseUrl}}/UserGuide.html#chart-panel)
+      * [Code Panel]({{baseUrl}}/UserGuide.html#code-panel)
+      * [Commits Panel]({{baseUrl}}/UserGuide.html#commits-panel)
+      * [Tool Bar]({{baseUrl}}/UserGuide.html#tool-bar)
+    * [Configuring a Repo to Provide Additional Data to RepoSense]({{baseUrl}}/UserGuide.html#configuring-a-repo-to-provide-additional-data-to-reposense)
+      * [Provide Data Using a json Config File]({{baseUrl}}/UserGuide.html#provide-data-using-a-json-config-file)
+      * [Provide Data Using `@@author` Tags]({{baseUrl}}/UserGuide.html#provide-data-using-author-tags)
+    * [Customizing the Analysis]({{baseUrl}}/UserGuide.html#customizing-the-analysis)
+      * [Customize Using Command Line Parameters]({{baseUrl}}/UserGuide.html#customize-using-command-line-parameters)
+      * [Customize Using csv Config Files]({{baseUrl}}/UserGuide.html#customize-using-csv-config-files)
+    * [Analyzing Multiple Repos]({{baseUrl}}/UserGuide.html#analyzing-multiple-repos)
+    * [Quickstart RepoSense with Netlify]({{baseUrl}}/UserGuide.html#quickstart-reposense-with-netlify)
+    * [Automate publishing of the report to GitHub Pages]({{baseUrl}}/UserGuide.html#automating-publishing-of-the-report-to-github-pages)
+    * [FAQ]({{baseUrl}}/UserGuide.html#faq)
+
+* <span class="lead">**Developer Guide**</span>
```

Comment:
Hmm I meant two completely separate site navs.
E.g. if you're on a user guide page, only the expanded user guide sitenav would show

Example of the userGuide site nav in markbind:
![Untitled](https://user-images.githubusercontent.com/3306138/78424561-0ee62f00-76a1-11ea-8134-1571e79b95dc.png)


The links in the sitenav seem to reflect the pagenav's links as well, you could instead include all the **pages** of the userGuide under the userGuide sitenav for inter page navigation, which is its intended use case. ( there aren't many pages for now as well but would be nice to have the structure layed out so we could always split it up later easily )

Reply:
Done. There were only two inter pages each for ug and dg.

Reply:
Is there a way such that if i only add a pageNav (without the siteNav), the content in the center won't be pushed towards the left of the entire page?

Reply:
not sure if it works, but you could try an empty an empty sitenav with just the `<navigation>` tag

Reply:
yep this works fine. Thanks so much for your help!